pkg/service/FluxService/FluxApplicationService.go

```
@@ -0,0 +1,420 @@
+		ReleaseName:   releaseName,
+		Namespace:     _namespace,
+		ClusterConfig: request.Config,
+	}
+
+	resourceTreeResponse, err := impl.common.BuildResourceTree(appDetailRequest, helmRelease)
+	if err != nil {
+		impl.logger.Errorw("error in building resource tree", "err", err)
+		return nil, appStatus, err
+	}
+	fluxAppTreeResponse = append(fluxAppTreeResponse, resourceTreeResponse)
+
+	return fluxAppTreeResponse, appStatus, nil
+}
+func (impl *FluxApplicationServiceImpl) buildFluxAppDetailForKustomize(request *FluxAppDetailRequest) ([]*bean.ResourceTreeResponse, *FluxAppStatusDetail, error) {
+	fluxAppTreeResponse := make([]*bean.ResourceTreeResponse, 0)
+
+	k8sClusterConfig := impl.converter.GetClusterConfigFromClientBean(request.Config)
+	restConfig, err := impl.k8sUtil.GetRestConfigByCluster(k8sClusterConfig)
+	if err != nil {
+		impl.logger.Errorw("error in getting the rest config", "error", err)
+		return nil, nil, err
+	}
+	resp, err := impl.k8sUtil.GetResource(context.Background(), request.Namespace, request.Name, GvkForKustomizationFluxApp, restConfig)
+	if err != nil {
+		impl.logger.Errorw("error in getting app manifest", "err", err)
+		return nil, nil, err
+	}
+	if resp == nil || resp.Manifest.Object == nil {
+		impl.logger.Errorw("app manifest's response or manifest object is nil", "err", err)
+		return nil, nil, err
+	}
+	appStatus, err := getKsAppStatus(resp.Manifest.Object, GvkForKustomizationFluxApp)
+	if err != nil {
+		impl.logger.Errorw("failed to get app status", "err", err)
+		return nil, nil, err
+	}
+	if !inventoryExists(resp.Manifest.Object) {
+		impl.logger.Errorw("Inventory is empty ", err)
+		return nil, appStatus, err
+	}
+
+	fluxK8sResourceList := make([]*client.ExternalResourceDetail, 0)
+	fluxHrList := make([]*FluxHr, 0)
+	impl.getFluxResourceAndFluxHrList(request, &fluxK8sResourceList, &fluxHrList, resp.Manifest.Object)
+	if len(fluxHrList) > 0 {
+		hrTreeResponse, _ := impl.getTreeForHrList(request, fluxHrList)
```

why are we ignoring the error here, if err here is meant to be ignnored then please log the error here just don't put return statement .